samples/subsys/fs/ext2/README.rst

```diff
@@ -0,0 +1,39 @@
+Ext2 Sample application
+#######################
+
+Overview
+********
+
+This sample app uses the Ext2 file system driver to mount an SDHC card connected over an SPI bus.
+
+Requirements
+************
+
+This sample requires SD card support and a microSD card formatted with Ext2 filesystem.
+Restrictions:
+
+* max file system size is:
+
+   .. code-block:: none
+
+      - 8MB for 1k blocks
+      - 32MB for 2k blocks
+      - 128MB for 4k blocks
+
+* file system has to be created with specific options (the implementation doesn't support all ext2 features)
+
+   .. code-block:: none
+
+      mkfs.ext2 /dev/<device> -b <block size> -I 128 -O ^ext_attr,^resize_inode,^dir_index,^sparse_super,^large_file
+
+
+Building and running
+********************
+
+The sample can be built for the `hifive_unmatched` board.
```

Comment:
Also in addition to above, this sample actually has a default offset to account for gpt, which the above does not mention (and setup of a gpt table is more complex than the above command)

Reply:
done